src/pages/quotes/Quotes.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Quotes = () => (
+  <section className="container">
+    <h1>
+      <q>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+        tempor incididunt
+      </q>
```

Comment:
[OPTIONAL]

- Instead of writing `Lorem ipsum ...`, it's better if you write one quote here so that if the recruiter will visit your website they found the website professional.